node/network/statement-distribution/src/lib.rs

```diff
@@ -1061,18 +1064,18 @@ async fn retrieve_statement_from_message<'a>(
 					let is_large_statement = message.is_large_statement();
 
 					match info.available_peers.entry(peer) {
```

Comment:
Sorry.

Another one.  :D 

```suggestion
					let is_new_peer = match info.available_peers.entry(peer) {
```

return `true` or `false` below and remove the check above. It doesn't really make sense to access the map twice.

Reply:
Fair, just found it slightly more readable this way, but in hindsight it probably isn't :-)